deploy/devdeploy.py

```diff
@@ -60,6 +100,10 @@ def devdeploy(profile: str, schema: str, stage: str, deployment: str):
     # Deploy the OpsCenter code into the stage.
     _copy_opscenter_files(cur, conn.schema, stage, deployment)
 
+    # Finish local setup by setting internal state to mimic a set-up app
+    # (e.g. materializes data, starts tasks)
+    _finish_local_setup(cur, conn.database, conn.schema)
```

Comment:
@JamesRTaylor can you put this behind a command line flag? There are instances where we don' twant this to happen...eg when testing the refresh tasks

Reply:
Will do in a follow up PR